src/app/pages/sign-out-confirm/sign-out-confirm-layout.tsx

```diff
@@ -28,11 +28,20 @@ export const SignOutConfirmLayout: FC<SignOutConfirmLayoutProps> = props => {
       <Box mx="loose" mb="extra-loose">
         <form onChange={form.handleChange} onSubmit={form.handleSubmit}>
           <Body>
-            When you sign out,
-            {whenWallet({
-              software: ` you'll need your Secret Key to sign back in. Only sign out if you've backed up your Secret Key.`,
-              ledger: ` you'll need to reconnect your Ledger to sign back into your wallet.`,
-            })}
+            <p>
+              When you sign out,
+              {whenWallet({
+                software: ` you'll need your Secret Key to sign back in. Only sign out if you've backed up your Secret Key.`,
+                ledger: ` you'll need to reconnect your Ledger to sign back into your wallet.`,
+              })}
+            </p>
+            <p style={{ marginTop: '24px', fontWeight: 'bold' }}>
```

Comment:
```suggestion
            <Text as="p" mt="loose" fontWeight="bold">
```

There's a text component you can use that supports the styling props